paragraph/graph/graph.cc

```diff
@@ -66,8 +66,8 @@ const std::vector<Instruction*> Graph::InstructionsPostOrder() const {
   return postorder;
 }
 
-absl::Status Graph::IsConnected() const {
-  RETURN_IF_ERROR(entry_subroutine_->IsConnected());
+absl::Status Graph::IsConnected(bool drop_disconnected) {
```

Comment:
This should be renamed to PruneDisconnected now

Reply:
By default it doesn't prune, only throws non-OK status, but you can prune if you overwrite the default. So for rename, the default should be changed, or even pruning should be mandated, otherwise the name would be confusing

Reply:
The name doesn't make sense. A method's name should tell what it does. You are mixing two things into one method where the name `IsConnected` does not make sense. If you really wish to keep this it as a single function, then name is something else, like VerifyConnected. It is a very common naming practice to have `Is...` method names be accessors only giving back a boolean for status.

Reply:
I renamed the function to `CheckIfConnected(bool drop_disconnected)` and make it private. So I added public `IsConnected()` and `DropDisconnected()` that work as could be expected.